consensus/core/src/block_manager.rs

```diff
@@ -332,6 +347,16 @@ impl BlockManager {
     }
 }
 
+// Result of trying to accept one block.
+enum TryAcceptResult {
+    // The block is accepted. Wraps the block itself.
+    Accepted(VerifiedBlock),
+    // The block is suspended. Wraps ancestors to be fetched.
+    Suspended(BTreeSet<BlockRef>),
+    // The block has been processed and exists in BlockManager or DagState.
+    Processed,
```

Comment:
nit: IIUC this case is when it has already been processed. However the name makes it sound like it just got processed as a result  of `try_accept`. I don't mind if you dont change the name but maybe specify in the comment. Other names I would suggest are `AlreadyAccepted` or `Skipped`

Reply:
The naming comes from Mingwei's earlier PR, but given that the code won't distinguish between an earlier accepted block and a suspended , probably the `Processed` offers this generic umbrella. But I'll refine a bit the comments yes.